packages/aws-cdk/test/integ/cli/common.bash

```diff
@@ -72,14 +72,14 @@ function prepare_fixture() {
     mkdir -p node_modules
 
     npm install \
-        @aws-cdk/core@^1 \
```

Comment:
`@^1` was added as part of this PR: https://github.com/aws/aws-cdk/pull/6472

This is because otherwise, the `npm` repo wrapper would fail: https://github.com/aws/aws-cdk/issues/6477.

Problem is that shortly after, the following fix was applied: https://github.com/aws/aws-cdk/pull/6482. 

The combination of these two actually negate one another because the `@^1` causes `lerna ls` to fail, bypassing the symlinking. We don't actually need this anymore.